### ISSUE TITLE
Add QWERTY(Ç) layout

### ISSUE THREAD

**Issue by Rafael2616** [florisboard/florisboard#1939]
this layout is very similar to QWERTY, for as presented, it has the character "Ç" located next to the letter "L" in the second row of letters, this design is applicable in PT-BR, I would like to implement it, thanks in advance!
![Screenshot_20220706-133050~2.png](https://user-images.githubusercontent.com/93414086/177600762-72eeb0b9-bf13-450a-a8cf-e835f520c23f.png)

**Comment by tsiflimagas** [comment]
> I would like to implement it

Do you mean you want to do it yourself and open a pull request, or to have it implemented by someone else? If it's the former, then you're welcome to go on with it. If it's the latter, then you can wait. It's a very small change, so it won't take long till someone implements it.

**Comment by Rafael2616** [comment]
the second option, since I'm not a developer. Thanks for the speed, I hope someone implements it in the near future, thanks!!

**Comment by tsiflimagas** [comment]
I just found out that this is already implemented as Catalan (QWERTY). I wonder if this name or QWERTY(Ç) is better to go.

**Comment by Rafael2616** [comment]
I just checked and it really is the same QWERTY(Ç) layout, well, here in Brazil it is known as QWERTY (Ç). and it's already familiar to about 200 million people and so it's used by Gboard, so I think the Ç at the end makes more sense...

**Comment by tsiflimagas** [comment]
Hmm yeah, sounds like right. @patrickgold what do you think about it? Should the layout be renamed?

**Comment by patrickgold** [comment]
Looked quite a bit in different sources and there is simply no standard, every OS and keyboard seems to use whatever it desires it seems. But I agree, renaming the layout makes things more clearer and Gboard has de facto set the standard name for this layout on Android due to its popularity.

If we begin thinking about layout names though shouldn't we also think about renaming other layouts too? It is just a pure mess by now and there is not even a proper naming standard within the relatively small amount of layouts that FlorisBoard supports...

**Comment by tsiflimagas** [comment]
Uh, you're right. Well, I don't even know if there's any standard to follow, other than following what Gboard does. And I don't know if it matters that much after all. If most keyboards use different names for many layouts, then users would have to do some manual searching through the layouts anyway, since you can't cover every keyboard's naming scheme out there. I guess that's why Gboard, Swiftkey and maybe others provide a visual representation of what layouts look like, so users don't rely much on the names. I have no strong opinion on how we should proceed here.

**Comment by Rafael2616** [comment]
You're right, I think the most consistent would be to follow Gboard, as it's the "default" for keyboards on most smartphones. I'll check other keyboards to see if I can find any similarities or not.
I don't know what to do either...

**Comment by klaurence** [comment]
I might've contributed code that would've added this layout, but the timing is unfortunate: Florisboard is currently being reworked such that it can't accept new layouts. The best solution right now probably is to wait.
